### DISCUSSION TITLE
Salt Edge not fetching recent transactions?

### DISCUSSION THREAD

**Discussion by Shazib** [firefly-iii/firefly-iii#5284] [Firefly III]
Hi,

I've got the spectre/saltedge importer all set up and they've given me Test access and i've run the importer successfully and imported the last 100 transactions which worked fine.

However now i'm trying to re-run it a couple weeks later to get the latest transactions but it doesn't seem to be updating them, even if i log into the saltedge dashboard i can see the 100 transactions there but they're from two weeks ago, and i can't see an option anywhere to trigger a delete+reload to get the latest 100.

do i have to remove and re-add the connection for this to work?

Thanks

**Comment by JC5** [comment]
You could try, I'm not sure.

**Comment by alexander-schoch-linuxdays** [comment]
Is there any solution to this? Reconnecting the account does not solve the problem in my case.
